src/modules/esl-toggleable/core/esl-toggleable.ts

```diff
@@ -8,6 +8,15 @@ import {DeviceDetector} from '../../esl-utils/environment/device-detector';
 import {DelayedTask} from '../../esl-utils/async/delayed-task';
 import {ESLBaseElement, attr, jsonAttr, boolAttr} from '../../esl-base-element/core';
 
+interface ESLShowRequestDetails {
```

Comment:
```suggestion
export interface ESLShowRequestDetails {
```
We can't predict if that's type will be useful from consumers side, so better let them asses it.